frontend/13f-app/src/components/SearchForm.js

```diff
@@ -1,22 +1,30 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
 import '../App.css'
-import { Button } from './Button'
-import SearchBox from './SearchBox'
+import './Button.css'
 import './MainSection.css'
 
 const divStyle = { float: 'left', padding: '20px', margin: '20px' }
 const date = new Date()
 const today = date.toISOString().substr(0, 10)
 
 function SearchForm () {
+  const [searchName, setSearchName] = useState('')
+  const [searchStartDate, setSearchStartDate] = useState(today)
+  const [searchEndDate, setSearchEndDate] = useState(today)
+  const handleNameChange = event => { setSearchName(event.target.value) }
+  const handleStartDateChange = event => { setSearchStartDate(event.target.value) }
+  const handleEndDateChange = event => { setSearchEndDate(event.target.value) }
+  const handleClick = (e) => console.log(searchName)
+
   return (
         <>
-            <SearchBox />
-            <div style={divStyle}>
-            <input type='date' className='date' defaultValue={today}></input>
-            <input type='date' className='date' defaultValue={today}></input>
-            </div>
-            <Button buttonStyle='btn--outline' buttonSize='btn--large'>SEARCH</Button>
+                <div style={divStyle}>
+                  <input type='text' placeholder='Company Name' className='search' value={searchName} onChange={handleNameChange}></input>
+                  <input type='date' className='date' defaultValue={today} value={searchStartDate} onChange={handleStartDateChange}></input>
+                  <input type='date' className='date' defaultValue={today} value={searchEndDate} onChange={handleEndDateChange}></input>
+                </div>
+                <Link to="/search"><button className="button" onClick={handleClick} type="submit">SEARCH</button></Link>
```

Comment:
```suggestion
                <Link to={
                  {
                    pathname: '/search',
                    state: { q: searchName, startDate: searchStartDate, endDate: searchEndDate }
                  }
                }><button className="button" onClick={handleClick} type="submit">SEARCH</button></Link</Link>
```

Reply:
It's worth noting that this isn't entirely good form for the design of the application. Normally we'd be able to allow the user to re-use the URL to retrieve the same search, but this solution doesn't leverage the URL at all. We'll need to refactor this